src/mozilla_sec_eia/library/generic_io_managers.py

```diff
@@ -19,3 +21,19 @@ def load_from_path(self, context: InputContext, path: UPath) -> pd.DataFrame:
         """Read parquet."""
         with path.open("rb") as file:
             return pd.read_parquet(file)
+
+
+class PickleUPathIOManager(UPathIOManager):
```

Comment:
I added this to save pickled asset outputs to GCS. This is needed because I separated the ex21 inference dataset creation from actually running the model, but the datasets take up too much space if they're saved locally.